pkg/database/asyncoperations.go

```diff
@@ -35,6 +35,13 @@ func NewAsyncOperations(uuid string, dbc cosmosdb.DatabaseClient, dbid, collid s
 	}, nil
 }
 
+func NewAsyncOperationsWithProvidedClient(uuid string, client cosmosdb.AsyncOperationDocumentClient) *asyncOperations {
```

Comment:
Better to return the exported interface rather than the unexported concrete type (throughout)
Might as well have NewAsyncOperations call NewAsyncOperationsWithProvidedClient like billing does (throughout if appropriate)

Reply:
done